.circleci/config.yml

```diff
@@ -353,9 +353,9 @@ jobs:
             # Wait for client to be setup
             echo "Waiting for beaconing before starting VPN client SCMPs"
             docker-compose exec -T useras4 /bin/bash -c \
-              "tail -n 0 -f /var/log/scion/* | awk '/Registered beacons/ {c++; if(c>=3) exit(0)}'"
+              "sleep 10; tail -n 0 -f /var/log/scion/* | awk '/Registered beacons/ {c++; if(c>=3) exit(0)}'"
```

Comment:
Why the sleep 10? (both here and line 451)

Reply:
The reason being that the services might still not have produces their first log, so /var/log/scion has no files.